packages/react-lowcode/src/codegen/generation/generators/template/template-resolver.ts

```diff
@@ -34,11 +34,11 @@ export default class TemplateResolver {
 
                 //find 'useGeneratedQuery' import and replace it with use'queryName's
                 const { listQueryName } = this._introspection ? getQueryNames(this._introspection, this._entity.getName()) : { listQueryName : undefined }
-                const hookName = queryHookName(listQueryName ?? '')
+                const hookName = 'useAllCustomersQuery' //queryHookName(listQueryName ?? '')//TODO:replace this with realn query name

```

Comment:
@rastiQ, is there any technical problem?

Reply:
it was not working and rasto found bug, that template resolver was not provided with introspection..fixed